packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1364,3 +2166,49 @@ A Role could either be assigned to the User directly or be assigned to a Group o
 | :------------------ | :----------------------------------------------------------------- |
 | `GROUP`             | Role was assigned to an admin Group of which the User is a member |
 | `USER`              | Role was assigned to the User directly                             |
+
+## Subscription object
+
+### Examples
+
+#### Sample Subscription
+
+```json
+{
+    "notificationType": "IWA_AGENT",
+    "channels": [
+        "email"
+    ],
+    "status": "subscribed",
+    "_links": {
+        "unsubscribe": {
+            "href": "http://rain.okta1.com:1802/api/v1/users/00uuk0UVgUXjkIbPL0g3/subscriptions/IWA_AGENT/unsubscribe",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "self": {
+            "href": "http://rain.okta1.com:1802/api/v1/users/00uuk0UVgUXjkIbPL0g3/subscriptions/IWA_AGENT"
+        }
+    }
+}
+```
+
+### Subscription properties
+
+The Subscription object defines several properties:
+
+| Property         | Description                                                  | DataType                                                        | Nullable   | Unique   | Read Only |
+| :--------------- | :----------------------------------------------------------- | :-------------------------------------------------------------- | :--------- | :------- | :-------- |
+| notificationType | Type of subscriptions                                        | [Notification Type](#notification-type)                         | FALSE      | TRUE     | TRUE      |
+| channels         | Array of sources that the user will get notification from    | array of [Channel](#subscription-channel)                       | FALSE      | FALSE    | TRUE      |
+| status           | Status of subscriptions                                      | `subscribed`, `unsubscribed`                                    | FALSE      | FALSE    | FALSE     |
```

Comment:
I though we decided the lack of channel in channel array should be considered as whether admin is subscribed or not @yuanxue-okta to keep me honest. Not sure for example what does it mean to have more than one channel in the array with a single subscribed or unsubscribed status. I don't see how that will be consumable by the caller

Reply:
We don't depend on the `channel` any more since we introduced `status` in the subscription